Userland/Libraries/LibTest/Macros.h

```diff
@@ -85,3 +85,9 @@ void current_test_case_did_fail();
             ::Test::current_test_case_did_fail();                                                               \
         }                                                                                                       \
     } while (false)
+
+#define FAIL(msg)                                                                  \
+    do {                                                                           \
+        ::AK::warnln("\033[31;1mFAIL\033[0m: {}:{}: {}", __FILE__, __LINE__, msg); \
+        ::Test::current_test_case_did_fail();                                      \
+    } while (false)
```

Comment:
```suggestion
#define FAIL(message)                                                                  \
    do {                                                                               \
        ::AK::warnln("\033[31;1mFAIL\033[0m: {}:{}: {}", __FILE__, __LINE__, message); \
        ::Test::current_test_case_did_fail();                                          \
    } while (false)
```

no need to abbreviate here - https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#names

>  Use full words, except in the rare case where an abbreviation would be more canonical and easier to understand.